corehq/apps/reports/views.py

```diff
@@ -794,7 +794,7 @@ def email_report(request, domain, report_slug, report_type=ProjectReportDispatch
     config.filters = filters
 
     report_id = None
-    if config.report:
+    if config.report and hasattr(config.report, '_id'):
```

Comment:
I dont think the built in reports do, so something like the Worker Activity Report doesnt have an id since it isnt stored as a report config in couch